app/assets/stylesheets/refills/_flashes.scss

```diff
@@ -9,9 +9,10 @@ $success-color: #e6efc2 !default;
   color: darken($color, 60%);
   display: block;
   font-weight: 600;
-  margin-bottom: $base-spacing / 2;
+  margin-bottom: 0;
   padding: $base-spacing / 2;
   text-align: center;
+  clear: both;
```

Comment:
Because the nav is floating the flash message div is behind it, and doesn't collide with it's bounds. By clearing it we go from this:

![withoutclear](https://cloud.githubusercontent.com/assets/8376505/8115414/b9c4b62a-1040-11e5-9407-9954be823bf0.png)

To this:

![withclear](https://cloud.githubusercontent.com/assets/8376505/8115417/bf5b6066-1040-11e5-90e1-2b8c007bb3be.png)

We can try for a different fix, but thought this looked pretty good (and allowed for easier fixing of the padding)